examples/example_pump_data_model/default.config.yaml

```diff
@@ -0,0 +1,18 @@
+# Only valid for this module, loads template variables from environment
+model_space: cdfTemplate
```

Comment:
This is a bit generic name? It would be good to have it more specific and maybe indicate with a comment how it should be overridden in config.yaml?